VHAR-7042
Paikkausurakoiden POT -ilmoituksen tekeminen

jarnovayrynen opened the pull request.

jarnovayrynen committed:
Korjaa ettei paikkaus-POT:it näy päällystys POT-listassa

jarnovayrynen committed:
Yhtenäistä pääll. ja paikkaus POT:ien perustietojen lomaketta

Mahdollista paikkauskohteen YHA-lähetys
Paikkauskohteen YHA-lähetyksessä yhaid:n oltava 99
Paranna ohjetta, että paikkausten kustannukset syötetään paikkauskohteet-välilehdellä (eikä päällystyskohteet)

jarnovayrynen committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-7048

jarnovayrynen committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-7048

jarnovayrynen committed:
Edistä paikkauspotin sanoman muodostamista ja testausta

jarnovayrynen committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-7048

jarnovayrynen committed:
Hae paikk.koht. kustannus POT-sanomaan

Paranna paikkaus-POT:in testejä

jarnovayrynen committed:
Yhtenäistä pääll. ja paikk. POT:ien alku- ja loppupvm kentät

jarnovayrynen committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-7048

jarnovayrynen committed:
Korjaa yksikkötestejä

jarnovayrynen committed:
Korjaa yksikkötestejä

jarnovayrynen committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-7048

jarnovayrynen committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-7048

jarnovayrynen committed:
Merge branch 'VHAR-7048' of github.com:finnishtransportagency/harja into VHAR-7048

jarnovayrynen committed:
Korjaa yksikkötestejä

jpluomajoki-solita commented on paikkauspot-yhaan.xml:
Mikä tämä tiedosto on ja miksi se on rootissa?

jpluomajoki-solita commented on src/clj/harja/kyselyt/paallystys.sql:
Onko tämä tarkoituksella tässä? Näyttää omaan silmään vähän oudolta sen tarkemmin tutkimatta

jarnovayrynen committed:
Korjaa yksikkötestejä

jarnovayrynen committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-7048

jarnovayrynen committed:
Poista vahingossa lisätty xml filu

jarnovayrynen commented on paikkauspot-yhaan.xml:
Hyvä pointti. Oli paikallisia testauksia varten ja jossain commitissa vahingossa valunut läpi. Poistan.

jarnovayrynen commented on src/clj/harja/kyselyt/paallystys.sql:
Joo, tässä oli bugi. Lisäämällä tuo IS NULL saadaan mutuaalisesti poissulkeva SQL-kysely, jotta POT-lomakkeet tulevat kälissä vain joko päällystykseen tai paikkaukseen, ei molempiin.

jarnovayrynen committed:
Merge branch 'develop' of github.com:finnishtransportagency/harja into VHAR-7048

jarnovayrynen committed:
Korjaa ylläpitokohdeosien liitosehto

Virheellisesti liitettiin ylläpitokohteen_aikataulun kautta. Oire oli että saman urakan eri kohteen
tietoja saattoi valua sähköpostitiedonantoon.
